packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -110,20 +110,21 @@ export const Map = () => {
   const onTileSetChange = (tileSetKey: string) => {
     setActiveTileSet(TILE_SETS.find(({ key }) => key === tileSetKey) as typeof TILE_SETS[0]);
   };
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
I think we could probably be more descriptive with our naming here, to indicate what `isOpen` refers to